cnv/cnv_users_guide/cnv-using-the-cli-tools.adoc

```diff
@@ -0,0 +1,21 @@
+[id="cnv-using-the-cli-tools"]
+= Using the CLI tools
+include::modules/cnv-document-attributes.adoc[]
+:context: using-the-cli-tools
+toc::[]
+
+There are two primary CLI tools used for managing resources in the cluster:
```

Comment:
A bit passive? How about [ProductName] has two primary CLI tools... (?)
I'm not familiar with CNV docs. So, in context what you have might be fine.